shared/editor/embeds/Grist.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import Frame from "../components/Frame";
+import Image from "../components/Image";
+import { EmbedProps as Props } from ".";
+
+function Grist(props: Props) {
+  const { matches } = props.attrs;
+  const sheetURL = matches[1];
+  return (
+    <Frame
+      {...props}
+      src={`${sheetURL}?embed=true`}
```

Comment:
Needs adjustment after [this](https://github.com/outline/outline/pull/3914/commits/09fba4b4aecc0b964efd637796c0504756a36ea5) :)
```suggestion
      src={props.attrs.href.replace(/(\?embed=true)?$/, "?embed=true")}
```